docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -92,104 +36,194 @@ is an iteration of the last one, hence it improves the decision made by the
 
 [discrete]
-[[dfa-classification-performance]]
-=== {classification-cap} performance
+[[ml-dfanalytics-classification-evaluation]]
+== 6. Evaluate and interpret the result
 
-As a rule of thumb, a {classanalysis} with many classes takes more time to run 
-than a binary {classification} process when there are only two classes. The 
-relationship between the number of classes and the runtime is roughly linear.
+include::ml-dfa-shared.asciidoc[tag=dfa-evaluation-intro]
 
-The runtime also scales approximately linearly with the number of involved 
-documents below 200,000 data points. Therefore, if you double the number of 
-documents, then the runtime of the analysis doubles respectively.
+You can measure how well the model has performed on your training data set by 
+using the `classification` evaluation type of the 
+{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The 
+{classification} evaluation offers the following metrics to evaluate the model 
+performance:
+
+* Multiclass confusion matrix
+* Area under the curve of receiver operating characteristic (AUC ROC) 
+
+The following metrics helps you interpret the analysis results:
+
+* {feat-imp}
+* `class_probability`
+* `class_score`
+
+
+[discrete]
+[[ml-dfanalytics-mccm]]
+=== Multiclass confusion matrix
+
+The multiclass confusion matrix provides a summary of the performance of the 
+{classanalysis}. It contains the number of occurrences where the analysis
+classified data points correctly with their actual class as well as the number
+of occurrences where it misclassified them.
+
+This is an example of a confusion matrix for a binary problem:
+
+image::images/confusion-matrix-binary.jpg[alt="Confusion matrix of a binary problem",width="75%",align="center"]
+
+It is a two by two matrix because there are only two classes (`true` and
+`false`). It shows the proportion of data points that is correctly identified as 
+members of a each class and the proportion that is misidentified.
+
+As the number of classes increases, the confusion matrix becomes more complex:
+
+image::images/confusion-matrix-multiclass.jpg[alt="Confusion matrix of a multiclass problem",width="100%",align="center"]
+
+This matrix contains the actual labels on the left side while the predicted 
+labels are on the top. The proportion of correct and incorrect predictions is 
+broken down for each class. This enables you to examine how the {classanalysis}
+confused the different classes while it made its predictions.
+
+
+[discrete]
+[[ml-dfanalytics-class-aucroc]]
+=== Area under the curve of receiver operating characteristic (AUC ROC)
+
+The receiver operating characteristic (ROC) curve is a plot that represents the 
+performance of the {classification} process at different predicted probability 
+thresholds. It compares the true positive rate for a specific class against the 
+rate of all the other classes combined ("one versus all" strategy) at the 
+different threshold levels to create the curve.
+
+.Example 
```

Comment:
> .Example 

This title is output such that it seems a bit squashed against the text. It might be sufficient to just use "For example, you have three classes..." instead.